Realm/Realm/Helpers/SerializationHelper.cs

```diff
@@ -99,6 +100,9 @@ internal static void PreserveSerializers()
             _ = new BsonArraySerializer();
 
             _ = new ObjectSerializer();
+
+            _ = new EnumerableInterfaceImplementerSerializer<IList<object>, object>();
+            _ = new ExpandoObjectSerializer();
```

Comment:
What code did you write where the `ExpandoObjectSerializer` was needed? As far as I know, `ExpandoObject` is not supported on Unity, so I'm kind of surprised we ended up with a code path that reaches its serializer, although it's possible that it's just a fallback and the bson library uses it for non-expando objects as well.

Reply:
I'll put the code in a repo for you to take a look.